open_academy/models/open_academy_session.py

```diff
@@ -0,0 +1,13 @@
+from odoo import models, fields
+
+
+class Session(models.Model):
+    _name = "openacademy.session"
+    _description = "Session"
+
+    name = fields.Char(string="Sessions", required=True)
+    start_date = fields.Date()
+    duration = fields.Float()
+    seats = fields.Integer(string="Number of seats")
+    instructor = fields.Many2one('res.partner', ondelete="set null")
```

Comment:
See the guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#38fields
```suggestion
    instructor_id = fields.Many2one('res.partner', ondelete="set null")
```